flocker/provision/_install.py

```diff
@@ -817,11 +817,17 @@ def task_enable_flocker_control(distribution):
             ),
             run("echo 'flocker-control-api\t4523/tcp\t\t\t# Flocker Control API port' >> /etc/services"),  # noqa
             run("echo 'flocker-control-agent\t4524/tcp\t\t\t# Flocker Control Agent port' >> /etc/services"),  # noqa
-            run_from_args(['service', 'flocker-control', 'start']),
+            #run_from_args(['service', 'flocker-control', 'start']).on(error=managed_errors_handler),
+            run_from_args(['service', 'flocker-control', action]),
```

Comment:
This is interesting. `action` is only used by the Ubuntu branch, because the Centos branch *always* does a restart (See `START = "restart"` above).

I wonder if it makes sense for Ubuntu to always do a restart too?